boost-common/src/main/java/io/openliberty/boost/common/boosters/MPHealthBoosterConfig.java

```diff
@@ -20,40 +20,23 @@
 import io.openliberty.boost.common.BoostLoggerI;
 import io.openliberty.boost.common.boosters.AbstractBoosterConfig.BoosterCoordinates;
 import io.openliberty.boost.common.runtimes.RuntimeI;
-import io.openliberty.boost.common.runtimes.TomeeRuntimeI;
 import io.openliberty.boost.common.config.ServerConfigGenerator;
 
 @BoosterCoordinates(AbstractBoosterConfig.BOOSTERS_GROUP_ID + ":mpHealth")
 public class MPHealthBoosterConfig extends AbstractBoosterConfig {
 
-    String libertyFeature = null;
-
     public MPHealthBoosterConfig(Map<String, String> dependencies, BoostLoggerI logger) throws BoostException {
-        String version = dependencies.get(getCoordinates(this.getClass()));
-        // if it is the 2.0 version = MP2.0 feature level
-        if (version.equals(MP_20_VERSION)) {
-            libertyFeature = MPHEALTH_10;
-        }
-    }
-
-    @Override
-    public String getLibertyFeature() {
-        return libertyFeature;
-    }
-
-    @Override
-    public void addServerConfig(ServerConfigGenerator config) {
-        // No config to write
+        version = dependencies.get(getCoordinates(this.getClass()));
     }
 
     @Override
     public List<String> getDependencies(RuntimeI runtime) {
     	List<String> deps = new ArrayList<String>();
-        if(runtime instanceof TomeeRuntimeI) {
-            deps.add("org.apache.geronimo:geronimo-health:1.0.1");
-            deps.add("org.apache.geronimo:geronimo-health-common:1.0.1");
-            deps.add("org.eclipse.microprofile.health:microprofile-health-api:1.0");
-        }
+        // if(runtime instanceof TomeeRuntimeI) {
+        //     deps.add("org.apache.geronimo:geronimo-health:1.0.1");
+        //     deps.add("org.apache.geronimo:geronimo-health-common:1.0.1");
+        //     deps.add("org.eclipse.microprofile.health:microprofile-health-api:1.0");
+        // }
         return deps;
```

Comment:
Will look for the follow-up commit including TomEE to see where this goes.   Understand it's not there now.